test/SILGen/keypaths_objc_optional_bridging.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-swift-emit-silgen -import-objc-header %swift_src_root/test/Inputs/ObjCOptionalRequirements.h %s | %FileCheck %s
+
+// REQUIRES: objc_interop
```

Comment:
What are the criteria for `REQUIRES: objc_interop`? Could tests like these simply `-enable-objc-interop`?

Reply:
Apparently linux is not happy with `-enable-objc-interop` when importing a header, even though the test passed when an Objective-C module was being imported instead.

Reply:
`REQUIRES: objc_interop` is enabled when the target runtime is built with ObjC interop, which is practically only on Apple platforms. `-enable-objc-interop`'s behavior is only maintained on those platforms, so I don't think it makes sense to try to test it when running tests for target platforms that don't have ObjC interop.